### ISSUE TITLE
Support Windows

### ISSUE THREAD

**Issue by archisgore** [polyverse/ropoly#31]
Some day in the deep future, we need to support this on Windows, so we can study windows ASLR.

**Comment by rsundahl** [comment]
The limitations should be limited to masche, and masche is able to compile on Windows, so we should already be there, or very close.

**Comment by archisgore** [comment]
Yup. That's where my mind is at too. I want to use the built-in disassembler first. So we move away from disasm as a first step. Then use masche and port to windows.

**Comment by archisgore** [comment]
In fact I'm exploring ways to get a build pipeline on a windows container ASAP. Maybe time to look at Azure DevOps Pipelines or whatever that's called.
